### PR TITLE
Fixed the Path

### PR DESCRIPTION
**Pull request by Jolly5**
Well... it seems like the path is correct now, except the little thing at Saturn.
I do however not know what the correct path there would be since this is the only currently route offered by Google Maps.
Other than that, I updated mainly the classes Waypoint, JsonData and Drawing.

### CODE REVIEW

File: app/src/main/java/mainburg/planetenweg/Drawing.java [hunk 1]
```diff
@@ -21,24 +21,30 @@
 public final class Drawing {
     private static final String TAG = Drawing.class.getSimpleName();
 
-    public static void drawPath(String result, GoogleMap map) throws JSONException {
-        for (String line : result.split("\n")) {
-            Log.v(TAG, result);
-        }
+    public static void drawPath(GoogleMap map, String... jsonresults) throws JSONException {
```
Comment (torkleyy) [inline app/src/main/java/mainburg/planetenweg/Drawing.java:24]: I figured you'd use this cheat :P

File: app/src/main/java/mainburg/planetenweg/directions/JsonData.java [hunk 2]
```diff
@@ -18,6 +18,7 @@
     private JsonData() {
     }
 
+    @Deprecated
     public static String getData() {
```
Comment (torkleyy) [inline app/src/main/java/mainburg/planetenweg/directions/JsonData.java]: We still need this method?

**Review by torkleyy — APPROVED** [review]

File: app/src/main/java/mainburg/planetenweg/directions/PathRequest.java [hunk 3]
```diff
@@ -48,13 +47,13 @@
...
 //                }
 //            }
-//        });
+//        });*/
 //    }
 //
 //    private String makeURL(LatLng[] waypoints) {
 //        try {
 //            StringBuilder urlString = new StringBuilder();
-//            urlString.append("https://maps.googleapis.com/maps/api/directions/json");
```
Comment (Jolly5) [inline app/src/main/java/mainburg/planetenweg/directions/PathRequest.java]: This change was not intended. My fault |-(

File: app/src/main/java/mainburg/planetenweg/directions/JsonData.java (hunk 2 above)
Comment (Jolly5) [reply app/src/main/java/mainburg/planetenweg/directions/JsonData.java]: No.

File: app/src/main/java/mainburg/planetenweg/Drawing.java (hunk 1 above)
Comment (Jolly5) [reply app/src/main/java/mainburg/planetenweg/Drawing.java:24]: First time I ever used this.

**Comment by torkleyy** [comment]
Thanks then!

**Pull request merged**